### ISSUE TITLE
[HL 25] Half-Life Bugs Steam Deck

### ISSUE THREAD

**Issue by ifohancroft** [ValveSoftware/steam-for-linux#10254]
I don't know how, but my girlfriend managed to install Half-Life on my Steam Deck a couple of days ago, but the game had problems (mostly with jumping not working) so she uninstalled it.

Today, I tried installing it again, but just opening the Half-Life library entry bugs the Steam Deck.

1. The install button doesn't work. Neither hitting A while the install button is hovered, nor touching the button on the screen.
2. After you've opened the Half-Life entry from the library, you can't restart the Steam Deck (or turn it off). It freezes during turning it off. Also, you can't even bring up the power off menu by holding the power button after opening the entry from the library.
3. Browsing around the library and the software, suddenly other buttons don't work, like pausing a game that's downloading, the Steam button doesn't work (the physical one), although ocasionally the one on screen also stops working.

If you do manage to restart the Steam Deck, it all works as expected again, as long as you don't open the Half-Life entry in the library. If you do, all the bugs start all over again.

**Comment by MetalClaw368** [comment]
Try installing the Half-Life game in the Steam Deck's Desktop Mode, and I was able to get it from there. I hope this helps!

**Comment by schmalpal** [comment]
This happens to me too. The issue isn't that you can't install HL, MetalClaw, it's that you literally can't view the HL game entry on the deck without it screwing everything up - power menu no longer works, steam button doesn't work, A button basically doesn't work, you can't perform any actions other than switching library views. You have to force a shutdown and restart by holding the power button. I previously had HL installed as well, so it has something to do with installing and then uninstalling it. Doesn't matter if I've purged every HL file from steamapps, it's still doing it in January 2024.

**Comment by cchulo** [comment]
this is still happening as of November 2024
I am on bazzite, so this is not a steam os unique issue. It worked fine installing half life for the very first time on a fresh image, but subsequent installs in game mode result in the same issue reported by OP

**Comment by KibSquib48** [comment]
I'm having this issue too, on a deck with SteamOS 3.6.20

**Comment by dslm4515** [comment]
I installed HL:Source on my SD. It loads the iconic Valve logo then locks up on what should be the main menu (no text). I launch it again from Desktop mode. Same issue.

A quick search online told me that I have to install HL2. I installed it in Desktop Mode. Now HL:Source launches fine and I can play. Then I go back to Gaming Mode. Same issue as before.

I go back in Desktop Mode. HL2 launches fine and plays fine. I close it. I launch HL:Source -- it freezes on the what should be the main menu, but just says "Half Life".

My SD is the LCD original with 512Gb running SteamOS 3.6.20.

**Comment by cchulo** [comment]
> I installed HL:Source on my SD. It loads the iconic Valve logo then locks up on what should be the main menu (no text). I launch it again from Desktop mode. Same issue.
> 
> A quick search online told me that I have to install HL2. I installed it in Desktop Mode. Now HL:Source launches fine and I can play. Then I go back to Gaming Mode. Same issue as before.
> 
> I go back in Desktop Mode. HL2 launches fine and plays fine. I close it. I launch HL:Source -- it freezes on the what should be the main menu, but just says "Half Life".
> 
> My SD is the LCD original with 512Gb running SteamOS 3.6.20.

this issue is relating to half life 25 anniversary. I recommend opening a new issue though half life source is no longer supported (check the half life 25th anniversary release notes)

**Comment by Spengreb** [comment]
Still an issue. I thought my steam deck was starting to break!

**Comment by usamaizm** [comment]
I'm still having this issue on Steam Deck SteamOS 3.7.15 Stable. I have to restart to resolve the issue but then the Steam Deck freezes during shutdown and I have to force it by holding the power button.

Edit: Forgot to mention it happens when pressing the install button. Nothing happens when you press install but instead the Steam Button and ... Button stop working until forced system restart. Even after restart, same symptoms persist with Half-Life being unable to be installed.

**Comment by cchulo** [comment]
this has also started happening with half life 2 btw after the anniversary update. The only workaround is to go to desktop mode and install from there. It is safe to then go back to game mode but only after the game has completed installation

**Comment by htaler** [comment]
I confirm the issue still exists today (stable release)
